src/AVXRunSumProcessor.cpp

```diff
@@ -26,21 +26,33 @@ void AVXRunSumProcessor::configure(const nlohmann::json& config, const int16_t*
   configure_internal_state_collection(config);
 
   int16_t memory_factors[16];
-  int16_t config_memory[3] = {config["memory_factor_plane0"],
-                              config["memory_factor_plane1"],
-                              config["memory_factor_plane2"]};
+  int16_t plane_memory_factors[3] = {config["memory_factor_plane0"],
+                                     config["memory_factor_plane1"],
+                                     config["memory_factor_plane2"]};
+  int16_t memory_divisors[16];
+  int16_t plane_memory_divisors[3] = {config["memory_divisor_plane0"],
+                                      config["memory_divisor_plane1"],
+                                      config["memory_divisor_plane2"]};
   int16_t scale_factors[16];
-  int16_t config_scale[3]  = {config["scale_factor_plane0"],
-                              config["scale_factor_plane1"],
-                              config["scale_factor_plane2"]};
+  int16_t plane_scale_factors[3]  = {config["scale_factor_plane0"],
+                                     config["scale_factor_plane1"],
+                                     config["scale_factor_plane2"]};
+  int16_t scale_divisors[16];
+  int16_t plane_scale_divisors[3] = {config["scale_divisor_plane0"],
+                                     config["scale_divisor_plane1"],
+                                     config["scale_divisor_plane2"]};
 
   for (int i = 0; i < 16; i++) {
-    memory_factors[i] = config_memory[plane_numbers[i]];
-    scale_factors[i] = config_scale[plane_numbers[i]];
+    memory_factors[i] = plane_memory_factors[plane_numbers[i]];
+    memory_divisors[i] = 0x7FFF / plane_memory_divisors[plane_numbers[i]];  // Need to adjust for AVX2 usage.
+    scale_factors[i] = plane_scale_factors[plane_numbers[i]];
+    scale_divisors[i] = 0x7FFF / plane_scale_divisors[plane_numbers[i]];  // Need to adjust for AVX2 usage.
```

Comment:
Just a comment. We should document the algorithm somewhere (the discussed saving pseudo-code). It is not immediately trivial to see this is (if I understand correctly) finding the inverse.